### PR TITLE
[TACHYON-1560] Rename tachyon.web.Utils to tachyon.web.WebUtils

### PR DESCRIPTION
**Pull request by ifcharming**
tachyon.web.WebUtils is better for readability since there are multiple Utils class.

https://tachyon.atlassian.net/browse/TACHYON-1560

### CODE REVIEW

File: servers/src/test/java/tachyon/web/WebUtilsTest.java [hunk 1]
```diff
@@ -21,111 +21,112 @@
 import org.junit.Test;
 
 /**
- * Unit tests for {@link Utils}.
+ * Unit tests for {@link WebUtils}.
  */
-public class UtilsTest {
+public class WebUtilsTest {
```
Comment (apc999) [inline servers/src/test/java/tachyon/web/WebUtilsTest.java]: can you mark this class `final`?

Comment (apc999) [inline servers/src/test/java/tachyon/web/WebUtilsTest.java]: p.s., this is a convention in Tachyon codebase that if possible, always mark non-base-classes `final`.

**Comment by apc999** [comment]
OK to test

**Comment by apc999** [comment]
Jenkins, test this please

File: servers/src/test/java/tachyon/web/WebUtilsTest.java (hunk 1 above)
Comment (ifcharming) [inline servers/src/test/java/tachyon/web/WebUtilsTest.java]: Done.

**Comment by ifcharming** [comment]
Jenkins, ok to test

**Comment by apc999** [comment]
Jenkins, test this please

**Comment by apc999** [comment]
+1 LGTM

thanks for submitting the patch!

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**